modules/util/css-builder/src/main/java/com/liferay/css/builder/CSSBuilder.java

```diff
@@ -61,72 +65,53 @@
 public class CSSBuilder implements AutoCloseable {
 
 	public static void main(String[] args) throws Exception {
-		Map<String, String> arguments = ArgumentsUtil.parseArguments(args);
-
-		List<String> dirNames = new ArrayList<>();
+		Map<String, String> argMap = new HashMap<>();
 
-		String dirName = GetterUtil.getString(
-			arguments.get("sass.dir"), CSSBuilderArgs.DIR_NAME);
+		for (int x = 0; x < args.length; x++) {
+			if (args[x].indexOf('=') != args[x].length() - 1) {
+				String[] arg = args[x].replace('=', ' ').split(" ");
 
-		if (Validator.isNotNull(dirName)) {
-			dirNames.add(dirName);
-		}
-		else {
-			for (int i = 0;; i++) {
-				dirName = arguments.get("sass.dir." + i);
+				if (arg.length == 2) {
+					if (arg[0].startsWith("sass.dir.")) {
+						arg[0] = "sass.dir";
+					}
 
-				if (Validator.isNotNull(dirName)) {
-					dirNames.add(dirName);
-				}
-				else {
-					break;
+					argMap.put(arg[0], arg[1]);
 				}
 			}
 		}
 
-		boolean appendCssImportTimestamps = GetterUtil.getBoolean(
-			arguments.get("sass.append.css.import.timestamps"),
-			CSSBuilderArgs.APPEND_CSS_IMPORT_TIMESTAMPS);
-		String docrootDirName = GetterUtil.getString(
-			arguments.get("sass.docroot.dir"), CSSBuilderArgs.DOCROOT_DIR_NAME);
-		boolean generateSourceMap = GetterUtil.getBoolean(
-			arguments.get("sass.generate.source.map"));
-		String outputDirName = GetterUtil.getString(
-			arguments.get("sass.output.dir"), CSSBuilderArgs.OUTPUT_DIR_NAME);
-
-		String portalCommonPath = arguments.get("sass.portal.common.path");
+		Collection<String> argsCollection = new ArrayList<>();
 
-		if (Validator.isNull(portalCommonPath)) {
-			portalCommonPath = arguments.get("sass.portal.common.dir");
+		for (Entry<String, String> entry : argMap.entrySet()) {
+			argsCollection.add(entry.getKey());
+			argsCollection.add(entry.getValue());
 		}
 
-		int precision = GetterUtil.getInteger(
-			arguments.get("sass.precision"), CSSBuilderArgs.PRECISION);
-		String[] rtlExcludedPathRegexps = StringUtil.split(
-			arguments.get("sass.rtl.excluded.path.regexps"));
-		String sassCompilerClassName = arguments.get(
-			"sass.compiler.class.name");
+		String[] argsArray = argsCollection.toArray(new String[0]);
 
-		try (CSSBuilder cssBuilder = new CSSBuilder(
-				appendCssImportTimestamps, docrootDirName, generateSourceMap,
-				outputDirName, portalCommonPath, precision,
-				rtlExcludedPathRegexps, sassCompilerClassName)) {
+		CSSBuilderArgs cssBuilderArgs = new CSSBuilderArgs();
 
-			cssBuilder.execute(dirNames);
+		Builder builder = JCommander.newBuilder();
+
+		builder.addObject(cssBuilderArgs);
+
+		JCommander commander = builder.build();
```

Comment:
About variable naming: Brian likes when variables have the same name of their type. So please call it `jcommander` and not `commander`.

Also, this seems different from what we already have in [ThemeBuilder](https://github.com/liferay/liferay-portal/blob/master/modules/util/portal-tools-theme-builder/src/main/java/com/liferay/portal/tools/theme/builder/ThemeBuilder.java#L54)... can you please copy from there?